ironfish/src/rpc/routes/fees/estimate.ts

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { ApiNamespace, router } from '../router'
+
+export type EstimateFeeRateRequest = { percentile: number }
+export type EstimateFeeRateResponse = { estimate: number }
+
+export const EstimateFeeRateRequestSchema: yup.ObjectSchema<EstimateFeeRateRequest> = yup
+  .object({
+    percentile: yup.number().defined(),
+  })
+  .defined()
+
+export const EstimateFeeRateResponseSchema: yup.ObjectSchema<EstimateFeeRateResponse> = yup
+  .object({
+    estimate: yup.number().defined(),
```

Comment:
This is a small nit but the result of "EstimateFeeRate" should be a fee rate right? The fee rate is the estimate.